io.openems.edge.heater.api/readme.adoc

```diff
@@ -0,0 +1,26 @@
+= Heater API 
+
+General Natures for heating devices. Heating devices are separated into 
+
+* `Chp` - Combined Heat and Power devices which can be monitored
```

Comment:
Format of readme.adoc is wrong. Make it analog to https://raw.githubusercontent.com/OpenEMS/openems/develop/io.openems.edge.ess.api/readme.adoc

Reply:
Good hint. Refactored, now it should fit better into the nature documentation.